pkg/server/grpc/option.go

```diff
@@ -82,3 +83,10 @@ func WithTLSConfig(cfg *tls.Config) Option {
 		o.tlsConfig = cfg
 	})
 }
+
+// LogDecision defines mapping of flag options to the logging decision.
+var LogDecision = map[string]grpc_logging.Decision{
```

Comment:
We can remove this I think

Reply:
Explained: https://github.com/thanos-io/thanos/pull/3361#discussion_r525897099